### PR TITLE
make some tasks optional

### PR DESCRIPTION
**Pull request by dcritch**
This PR will add a check for certain environment variables to optionally enable/disable some of the periodic tasks (watchlist, escalations, bzs)

### CODE REVIEW

File: dashboard/src/t5gweb/cache.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def get_cases(cfg):
 def get_escalations(cfg):
     '''Get cases that have been escalated from Smartsheet'''
     cases = libtelco5g.redis_get('cases')
-    if cases is None:
+    if cases is None or cfg['smartsheet_access_token'] is not None or cfg['smartsheet_access_token'] != '':
```
Comment (adhil0) [inline dashboard/src/t5gweb/cache.py:64]: Would we want the last two clauses to be inversed? Ex: `cfg['smartsheet_access_token] is None` so if the variable isn't defined the redis variable is set to None and the function returns

File: dashboard/src/t5gweb/cache.py [hunk 2]
```diff
@@ -229,6 +229,10 @@ def get_cards(cfg, self=None, background=False):
 def get_watchlist(cfg):
 
     cases = libtelco5g.redis_get('cases')
+    if cases is None or cfg['watchlist_url'] is not None or cfg['watchlist_url'] != '':
```
Comment (adhil0) [inline dashboard/src/t5gweb/cache.py:232]: See above

File: dashboard/src/t5gweb/cache.py [hunk 3]
```diff
@@ -290,9 +294,10 @@ def get_bz_details(cfg):
     '''Get details about Bugzillas from API'''
     logging.warning("getting additional info via bugzilla API")
     bz_dict = libtelco5g.redis_get('case_bz')
-    if bz_dict is None:
+    if bz_dict is None or cfg['bz_key'] is None or cfg['bz_key'] == '':
```
Comment (adhil0) [inline dashboard/src/t5gweb/cache.py:297]: See above

**Review by adhil0 — CHANGES_REQUESTED** [review]

File: dashboard/src/t5gweb/cache.py (hunk 3 above)
Comment (dcritch) [reply dashboard/src/t5gweb/cache.py:297]: I actually got it right on this one :)

**Comment by dcritch** [comment]
@adhil0 good catch! I was going back and forth between the affirmative test in the taskmgr vs. the negative tests in the cache. thanks!

**Review by adhil0 — APPROVED** [review]
Thanks, looks great!

**Pull request merged**